src/util/getExperienceName.ts

```diff
@@ -0,0 +1,41 @@
+import { BlobServiceClient, ContainerClient } from "@azure/storage-blob";
+
+const sasToken =
```

Comment:
the sas token is used in other files. I suggest you create a separate file for all the constants that are used throughout the code so they can be easily reused